cpp/ql/test/library-tests/dataflow/dataflow-tests/dataflow-ir-consistency.expected

```diff
@@ -25,6 +25,7 @@ postWithInFlow
 | test.cpp:391:10:391:13 | memcpy output argument | PostUpdateNode should not be the target of local flow. |
 | test.cpp:400:10:400:13 | memcpy output argument | PostUpdateNode should not be the target of local flow. |
 | test.cpp:407:10:407:13 | memcpy output argument | PostUpdateNode should not be the target of local flow. |
+| test.cpp:1045:9:1045:11 | memset output argument | PostUpdateNode should not be the target of local flow. |
```

Comment:
What is the consistency check failure telling us?

Reply:
The local flow step relation should ideally not target a `PostUpdateNode` since `PostUpdateNode`s are only supposed to be targeted by `storeStep`s (to model writes to a field) and interprocedurally (when flowing out of a write to a parameter).

However, flow through modelled function are implemented as local flow steps, so a call such as:
```cpp
memset(buffer, x, n);
```
which (if we had the source for `memset` and no model for it) would flow as follows (roughly):
```
x -> parameter 1 of memset -> parameter 0 of memset -> buffer [post update]
```
where the final step from `parameter 0 of memset` to `buffer [post update]` would be an interprocedural step.

But since we have a model for `memset` that (after this PR) says that there's flow from `x` to `buffer [post update]` directly, we get that step as part of the local flow step relation.

This isn't a problem. There's nothing wrong with `PostUpdateNode`s being targeted by the local flow step relation in practice. It was added a long time ago before languages started to have this kind of flow.

Reply:
Perhaps we should turn off this consistency check then, if it's not the case that it should always hold?

Reply:
We should definitely tune it, yeah. I'll see if I can figure out how to tune it to do what we want.